crates/freeze/src/datasets/logs.rs

```diff
@@ -284,5 +297,226 @@ async fn logs_to_df(
         cols.push(Series::new("chain_id", vec![chain_id; n_rows]));
     }
 
+    if let Some(decoder) = decoder {
+        // Write columns even if there are no values decoded - indicates empty dataframe
+        let chunk_len = block_number.len();
+        if event_cols.is_empty() {
+            for name in decoder.field_names().iter() {
+                cols.push(Series::new(name.as_str(), vec![None::<u64>; chunk_len]));
```

Comment:
instead of u64 should use whatever the decoded type would be. that way it'll be possible to concatenate the output chunk parquet files since theyll have the same type

Reply:
the underlying type isn't knowable at compile time, there might be a way to create a dynamically typed vec in rust I'm unaware of but this path, very open to suggestions